### DISCUSSION TITLE
Enabling revisions on existing database breaks subscription so no documents is subscribed on

### DISCUSSION THREAD

**Discussion by marcuslindblom** [ravendb/ravendb#15955] [Q&A] [answered]
To test, create a new database without revisions, add documents, add a subscription `from @all_docs (Revisions = true)`. Test subscription, no documents are listed. Modify one or more documents, test subscription again. Documents are listen. Is this a bug? It produces a problem for us when we enable revisons and we need to patch all documents to make the subscription work again.

**Comment by ayende** [comment, marked as answer]
`Revisions = true` means that we are working the revisions only, not the documents.
That is by design.

Note that only the documents with revisions are there, not document that weren't modified.

**Reply by marcuslindblom** [reply]
Ok, I see. We added revisions to be able to see deletes but that got us into other issues. Is it possible to get deletes on a subscription without revisions? I guess not 🤔

**Reply by georgiosd** [reply]
Maybe add a `Deleted` property that the subscription catches and then actually deletes the doc?

**Reply by marcuslindblom** [reply]
Thats how we do it today, you mean I can delete it in the subscription as well?

**Reply by georgiosd** [reply]
Of course, you're not limited in any way I am aware of in the subscription context.
